test/e2e/distributedtracing/distributedtracing_test.go

```diff
@@ -91,10 +91,11 @@ func TestServerSpan(t *testing.T) {
 			attribute.Int("net.host.port", port),
 			attribute.String("http.method", "POST"),
 			attribute.String("http.scheme", "http"),
-			attribute.String("http.flavor", "1.1"),
+			attribute.String("net.protocol.version", "1.1"),
```

Comment:
`http.flavor`->`net.protocol.version`

Reply:
Per [this](https://opentelemetry.io/blog/2023/http-conventions-declared-stable/#common-attributes-across-http-client-and-server-spans), `net.protocol.version` is now `network.protocol.version`.

Reply:
Yeah, I saw. But that's not part of this version bump. Maybe it'll come as an additional breaking change in a future release ..